kit/Kit.cpp

```diff
@@ -748,7 +748,7 @@ Document::Document(const std::shared_ptr<lok::Office>& loKit, const std::string&
     , _docId(docId)
     , _url(url)
     , _obfuscatedFileId(Uri::getFilenameFromURL(docKey))
-    , _queue(std::make_shared<KitQueue>())
+    , _queue(new KitQueue())
```

Comment:
If we always create a `KitQueue` and it seems never reset or re-assign it, perhaps we don't need a pointer at all. This removes an indirection and the many `_queue &&`.